### PR TITLE
Review edits

### PR DESCRIPTION
**Pull request by mhideo**
1st Pass of replacement edits. Still need to do another pass.

### CODE REVIEW

**Comment by mhideo** [comment]
@morgan and @tjdett can you please review this so i can merge this to develop.

**Comment by morgan** [comment]
@mhideo ^ @ephox-mogran

File: cloud-deployment-guide/editor-and-features.md [hunk 1]
```diff
@@ -18,12 +18,22 @@ All Tiny Cloud accounts include a free, pre-configured cloud image proxy service
...
 
 ```js
-<script src="{{ site.cdnurl }}?apiKey=your_API_key"></script>
+<script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
 ```
 
+||Value|Description|
+|---|---|---|
+|1|no-api-key|Replace with your api key|
+|2|origin=|A `referrerpolicy` specifies how much of the current page's URL is sent in the `Referer` header when the browser fetches page resources (for example, the TinyMCE editor). The use of the `Referer` header ensures API keys are only used on domains registered to their owners. We only care about the domain portion however (similar to the operation of `Origin` header), so for improved performance and privacy always set the `referrerpolicy` to `origin` when requesting Tiny resources.|
```
Comment (lnewson) [inline cloud-deployment-guide/editor-and-features.md]: Couple minor typos with "Referrer" here by the looks?

Comment (ephox-mogran) [reply cloud-deployment-guide/editor-and-features.md]: Referer is the official (incorrect) spelling. ReferrerPolicy chose not to preserve that misspelling.

Comment (ephox-mogran) [reply cloud-deployment-guide/editor-and-features.md]: > Note that Referer is actually a misspelling of the word "referrer". The Referrer-Policy header does not share this misspelling.

(Source: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy)

Comment (lnewson) [reply cloud-deployment-guide/editor-and-features.md]: I'd never noticed that, but good to know :+1:

**Pull request closed without merging**